src/screens/EthToSub.tsx

```diff
@@ -200,6 +214,24 @@ const EthToSub = ({ className, ethProvider } : Props) => {
 	);
 };
 
+function toEthBalance(v: BigInt) {
```

Comment:
Could we move these 2 to `util` as well? I feel like if there ever is a new page, we'll need them there as well :)

Reply:
Updated.